packages/mcfunction/src/colorizer/macro.ts

```diff
@@ -0,0 +1,32 @@
+import * as core from '@spyglassmc/core'
+import type {
+	MacroGapNode,
+	MacroKeyNode,
+	MacroSignNode,
+	MacroVariableNode,
+} from '../node/macro'
+
+export const macroChild: core.Colorizer<
+	MacroGapNode | MacroVariableNode | MacroSignNode
+> = (node, ctx) => {
+	const color = node.type === 'mcfunction:macro/gap'
+		? 'string'
+		: 'literal'
```

Comment:
Not sure `literal` is a good name, and there are some duplicate color mappings already. `macro_sign` could work I guess, as that's also how you call the node. Could be sth sth string interpolation marker as well, but that's quite long.

I think the other color names you are using somewhat fit.

Reply:
Color names that resolve to the same color, aren't all that necessary, become a little unruly if we have too many

Reply:
It's not a matter of naming it's just a matter of using built-in colorizer colors (didn't feel the need to add more duplicates). This isn't referenced anywhere else so I don't think it's an issue